clients/StateChannelWallet.ts

```diff
@@ -4,7 +4,7 @@ import {
   type HTLCStruct,
   type StateStruct,
   type NitroSmartContractWallet,
-} from "../typechain-types/Nitro-SCW.sol/NitroSmartContractWallet";
+} from "../typechain-types/contracts/Nitro-SCW.sol/NitroSmartContractWallet";
```

Comment:
Previously the `contracts` subdirectory was not included in this filepath. I'm not sure what changed, but now it is needed